src/components/VHeader/VHeader.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div
+    class="fixed flex px-8 align-center z-40 w-full bg-white"
```

Comment:
```suggestion
    class="sticky top-0 flex px-8 align-center z-40 w-full bg-white"
```

This should be preferable for the header, then we can remove the padding from the 'embedded' layout and the 'border appearing scroll' behavior should be smoother.